### PR TITLE
Add simpler explanations where I got confused originally reading this.

### PR DESCRIPTION
**Pull request by ddustin**
Essentially I added the simple explanation for some things next to the more complex ones originally used. If I had these in there when I first came upon this document, I would have had a much easier time digesting it.

Hopefully these changes help the next peson who reads it.

### CODE REVIEW

**Comment by luke-jr** [comment]
@sipa

File: bip-0032.mediawiki [hunk 1]
```diff
@@ -64,6 +65,8 @@ In what follows, we will define a function that derives a number of child keys f
 
 We represent an extended private key as (k, c), with k the normal private key, and c the chain code. An extended public key is represented as (K, c), with K = point(k) and c the chain code.
 
+When deriving child keys, a 'hardened' child key can only be generated using a private key. This provides security advantages but reduces the usefulness of HD key derivation. It is typically used to separate 'accounts' from each other.
```
Comment (ProofOfKeags) [inline bip-0032.mediawiki:68]: ACK this change.

File: bip-0032.mediawiki [hunk 2]
```diff
@@ -51,10 +51,11 @@ Addition (+) of two coordinate pair is defined as application of the EC group op
 Concatenation (||) is the operation of appending one byte sequence onto another.
 
 As standard conversion functions, we assume:
-* point(p): returns the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p.
+* point(p): returns the public key for p. This is the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p.
```
[the remark this answers is not in the document]
Comment (ProofOfKeags) [reply bip-0032.mediawiki:54]: Yeah I'm in agreement with @sipa here. Points are used as public keys but aren't equivalent to them in certain use contexts.

File: bip-0032.mediawiki (hunk 1 above)
Comment (apoelstra) [reply bip-0032.mediawiki:68]: I'd prefer the last sentence be dropped. Given the weird security properties of "normal" derivation, I think we should word things as though hardened derivation were the ordinary case, and non-hardened derivation is the one that has a sentence explaining why somebody would want to use it.

**Comment by ddustin** [comment]
> @ddustin mind updating per the new feedback?

Absolutely, will do. Traveling at the moment 👍

**Review by 5twelve — APPROVED** [review]

File: bip-0032.mediawiki (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddustin) [reply bip-0032.mediawiki:54]: How about this rewording:

"point(p): returns the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p ***(similar to making public key)***."

File: bip-0032.mediawiki (hunk 1 above)
Comment (ddustin) [reply bip-0032.mediawiki:68]: How about this wording to make it sound more ordinary:

"When deriving child keys, a 'hardened' child key can only be generated using a private key. This provides security advantages ***and prevents adversarial public key tracing***. It is typically used to separate 'accounts' from each other."

And another sentence added for non-hardened keys:

"***This non-hardened result is typically used by a server to continually generate receive addresses without the ability to spend funds.***"

File: bip-0032.mediawiki [hunk 3]
```diff
@@ -51,10 +51,11 @@ Addition (+) of two coordinate pair is defined as application of the EC group op
 Concatenation (||) is the operation of appending one byte sequence onto another.
 
 As standard conversion functions, we assume:
-* point(p): returns the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p.
+* point(p): returns the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p (similar to making public key).
```
Comment (murchandamus) [inline bip-0032.mediawiki:54]: There seems to be a word missing here:

```suggestion
* point(p): returns the coordinate pair resulting from EC point multiplication (repeated application of the EC group operation) of the secp256k1 base point with the integer p (similar to making a public key).
```

**Comment by jonatack** [comment]
@ddustin do you plan to respond to the BIP author's feedback of 3 months ago? Let's give this until the end of this month and then either close it or a BIP Editor may update it with the feedback. Thanks!

**Comment by murchandamus** [comment]
This pull request has not seen activity in several months, so I’m closing it for now. Please ping here to reopen it, if you want to continue working on it.

**Comment by jonatack** [comment]
Picked up by https://github.com/bitcoin/bips/pull/1903.
